### ISSUE TITLE
Can not DescribeInstanceCreditSpecifications

### ISSUE THREAD

**Issue by leandroscarneiro** [Corymbia/eucalyptus#115]
Hello, I'm trying to use Terraform with Eucalyptus, but after create an instance, it tries to "DescribeInstanceCreditSpecifications", and fails.
I've notice that might not be implemented.

**Comment by sjones4** [comment]
We do not currently support this action but it looks like it should be ok to add support in 4.4.5. If it turns out to be too complex for a maintenance release then we'll move it to 5.0.

**Comment by sjones4** [comment]
In case you are able to test if having a stub implementation for `DescribeInstanceCreditSpecifications` addresses this issue you can use this snapshot release of 4.4.5 work in progress:

http://downloads.eucalyptus.cloud/software/eucalyptus/snapshot/4.4/rhel/7/x86_64/eucalyptus-release-4.4-2.7.as.el7.noarch.rpm

Note that this should be used for testing only, snapshot releases should not be used in production and do not support upgrade to/from released versions.

The release rpm installs the following configuration, which you could do manually instead:
```
# cat /etc/yum.repos.d/eucalyptus.repo 
[eucalyptus]
name=Eucalyptus 4.4 - $basearch
baseurl=http://downloads.eucalyptus.cloud/software/eucalyptus/snapshot/4.4/rhel/7/x86_64/
gpgkey=https://downloads.eucalyptus.cloud/software/gpg/RPM-GPG-KEY-eucalyptus-release-as
       https://downloads.eucalyptus.cloud/software/gpg/RPM-GPG-KEY-eucalyptus-release
gpgcheck=1
enabled=1
```
Once the snapshot repository is configured you would use the regular install steps:

http://docs.eucalyptus.cloud/eucalyptus/4.4.4/index.html#install-guide/eucalyptus.html

We may still push this to 5.0, so let us know if the fix is useful for you in 4.4.5.

**Comment by sjones4** [comment]
A stub implementation for this action is present in 5.0 (Corymbia/eucalyptus@387d1b8dd86f25a546c9a20dc8c2f3e89650a06a) and has N4J test coverage in `TestEC2Api`.
